queueUI/mainwindow.py

```diff
@@ -32,6 +33,25 @@ def __init__(self):
         settings.restoreWindowGeometry(self, "mainwindow_geometry")
         print("Settings are saved in:", settings.fileName())
 
+        self.RM = REManagerAPI()
+        self.runEngineOpenButton_3.clicked.connect(self.openRunEngine)
+        self.runEngineCloseButton_3.clicked.connect(self.closeRunEngine)
+
+    # RE Function:
+    def openRunEngine(self):
+        self.RM.environment_open()
+        self.RM.wait_for_idle()
+
+        print(f"status={self.RM.status()}")
+
+    def closeRunEngine(self):
+        self.RM.environment_close()
+        self.RM.wait_for_idle()
+        print(f"status={self.RM.status()}")
```

Comment:
Do not print the whole RM status dictionary, only print the relevant key/value pair (`worker_environment_exists`?)

Reply:
Change made.

Reply:
Not seeing that yet.